go.sum

```diff
@@ -9,6 +9,7 @@ cloud.google.com/go v0.46.2/go.mod h1:a6bKKbmY7er1mI7TEI4lsAkts/mkhTSZK8w33B4RAg
 cloud.google.com/go/bigquery v1.0.1/go.mod h1:i/xbL2UlR5RvWAURpBYZTtm/cXjCha9lbfbpx4poX+o=
```

Comment:
I don't see an associated go.mod with this - was it committed by mistake?

Reply:
cleaned up `go.sum`.